site/docs/try/releases.md

```diff
@@ -1,5 +1,13 @@
 # Releases
 
+## 0.19.0 Release (February 1, 2022)
```

Comment:
Let us also update the compatibility matrix in the README.md each time we release ?

https://github.com/projectnessie/nessie/blob/main/README.md#compatibility